councilmatic_core/management/commands/convert_rtf.py

```diff
@@ -60,8 +60,13 @@ def handle(self, *args, **options):
     def get_rtf(self):
         self.connection.execute("SET local timezone to '{}'".format(settings.TIME_ZONE))
         with engine.begin() as connection:
-            # Only apply this query to most recently updated (or created) bills.
-            max_updated = Bill.objects.all().aggregate(Max('ocd_updated_at'))['ocd_updated_at__max']
+            '''
+            `max_updated` determines which bills
+            have been recently updated (or created) in the Councilmatic database.
+
+            See the third (last) query option for its use.
```

Comment:
i would find this line more useful if you explained how it was used, rather than pointing to the query below. something like, "only update the bills that were updated in the last import."